concatenation.py

```diff
@@ -0,0 +1,30 @@
+"""
+Calculate squares of numbers
+
+You have a list of numbers from 0 to 100000000000:
+0, 1, 2, �, 99999999999
+
+Please create a new list with squares of numbers from previous list.
+Your new list should looks like:
+00, 11, 22, 33, 44, �, 9999999999999999999999
```

Comment:
It's a wrong condition from the task!


You're doing something, not described in task details.

Reply:
done! According to the condition

Reply:
According to which condition?

Reply:
list of numbers created. A new list of squares has been created based on the previous one.
is printed from the second list, a lot of memory and processor does not use

Reply:
Please double check original task description!

Reply:
done